### PR TITLE
enable event scheduler

### PR DESCRIPTION
**Pull request by Lafunamor**
This PR allows users to enable the event scheduling feature of mariadb.
https://mariadb.com/kb/en/mariadb/events/

### CODE REVIEW

File: jobs/mysql/spec [hunk 1]
```diff
@@ -189,7 +189,10 @@ properties:
   cf_mysql.mysql.log_queries_not_using_indexes:
     default: false
     description: "Queries that don't use an index, or that perform a full index scan where the index doesn't limit the number of rows, will be logged to the slow query log."
-
-  cf_mysql.mysql.startup_timeout:
+  cf_mysql.mysql.event_scheduler:
+    default: on
```
Comment (robdimsdale) [inline jobs/mysql/spec]: Should the default here be `on` or `off`?

**Comment by robdimsdale** [comment]
This mostly looks fine - one small comment about the default state.

How does one configure this scheduler? I'm mostly asking so we have a conversation here that we can reference if there are questions in the future; we don't need this feature.

One thing we'd like to point out is that it's possible that when you schedule an event, it is executed on all nodes in the cluster, which can lead to some unexpected behavior. You do need the "Process" privilege to do it, so that does limit the scope.

**Comment by Lafunamor** [comment]
You're right. Default state should be off. I've changed this.

**Comment by robdimsdale** [comment]
Couple of things that we found with this PR:

yaml maps `on` and `off` to boolean values, not strings, which means they come out the other side of our ERB templates as `true/false` which `my.cnf` does not accept. We needed to quote the spec default as `"off"`.

The field was in the wrong place in `my.cnf` - it needed to be moved under `[mysqld]`.

**Pull request merged**